client/src/components/App.jsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-unused-vars */
```

Comment:
You shouldn't disable linting rules just because. The rules are trying to show you something. App here appears to not be necessary, and if it was for some reason, it could be a functional-stateless component.

Reply:
I forgot about these, thank you!